src/main/java/soot/jimple/toolkits/callgraph/OnFlyCallGraphBuilder.java

```diff
@@ -550,6 +550,25 @@ public void addType(Local receiver, Context srcContext, Type type, Context typeC
           }
         } else {
           VirtualCalls.v().resolve(type, receiver.getType(), site.subSig(), site.container(), targetsQueue, appOnly);
+          if(!targets.hasNext()) {
+            /* In the situation where we find nothing to resolve an invoke to in the first call, this
+             * might be because the type for the invoking object is a abstract class and the method is
+             * declared in a parent class. In this situation, when the abstract class has no classes
+             * that extend it in the scene, resolve would not find any targets for the invoke, even
+             * if the parent contained a possible target. 
+             * 
+             * This may have been by design since without a concrete class, we have no idea if the 
+             * method in the parent class is override. However, the same could be said for any non
```

Comment:
How could you possibly override a private method?

Reply:
You can't. The comment says this corner case only applies to **non private methods**. It is using the same resolution procedure you would use to resolve any method to a method in a parent class normally so there should not be any issue here.